bin/templates/cordova/lib/build.js

```diff
@@ -409,7 +457,31 @@ function parseOpts(options, resolvedTarget) {
             console.warn('Build option \'' + options[i] + '\' not recognized (ignoring).');
         }
     }
+    if (packageArgs.keystore && packageArgs.alias) {
+        var keystore = path.relative(ROOT, path.resolve(packageArgs.keystore));
+        ret.packageInfo = new PackageInfo(keystore, packageArgs.alias, packageArgs.keystorePassword,
+            packageArgs.password, packageArgs.keystoreType);
+    } else if (fs.existsSync(buildConfig)) {
+        console.log('Reading build config file: '+ buildConfig);
+        var config = JSON.parse(fs.readFileSync(buildConfig, 'utf8'));
+        if (config.android && config.android[ret.buildType]) {
+            var androidInfo = config.android[ret.buildType];
+            if (androidInfo.keystore && androidInfo.alias) {
+                var configDir = path.dirname(buildConfig);
+                var keystorePath = path.relative(ROOT, path.join(configDir, androidInfo.keystore));
+                ret.packageInfo = new PackageInfo(keystorePath, androidInfo.alias,
+                    androidInfo.keystorePassword, androidInfo.password, androidInfo.storeType);
```

Comment:
A bit inconsistent to have "keystorePassword" and "storeType" (should be storePassword, or keystoreType). I think the [gradle terms](http://stackoverflow.com/questions/18328730/how-to-create-a-release-signed-apk-file-using-gradle) for these make sense, so might as well use them?